### PR TITLE
It makes no sense to try using invalid or expired link again

### PR DESCRIPTION
**Pull request by mkljczk**

### CODE REVIEW

**Comment by ykzts** [comment]
ref https://github.com/tootsuite/mastodon/pull/4509#discussion_r131385758

File: config/locales/en.yml [hunk 1]
```diff
@@ -234,7 +234,7 @@ en:
     resend_confirmation: Resend confirmation instructions
     reset_password: Reset password
     set_new_password: Set new password
-    invalid_reset_password_token: Password reset link is invalid or expired. Please try again.
+    invalid_reset_password_token: Password reset link is invalid or expired. Have you already used it?
```
Comment (nullkal) [inline config/locales/en.yml]: As per https://github.com/tootsuite/mastodon/blob/8d6c3cd48ae4f96752ff6b698bc7244d97aa9a27/config/initializers/devise.rb#L230, password reset token expires in 6 hours without consume it. I think "Please request a new one." is better.

**Review by nullkal — CHANGES_REQUESTED** [review]

**Comment by ykzts** [comment]
resolve conflicts.

**Comment by m4sk1n** [comment]
@ykzts done

**Review by ykzts — APPROVED** [review]

**Pull request merged**